client/my-sites/plan-features/item.jsx

```diff
@@ -27,15 +28,25 @@ export default function PlanFeaturesItem( {
 		onMouseLeave( event.currentTarget, description );
 	};
 
+	const mouseEvents = {
+		onMouseEnter: handleOnMouseEvent,
+		onMouseLeave: handleOnMouseLeave
+	};
+	const hoverOnRow = (
+		abtest( 'plansDescriptions' ) === 'ascendingPriceEagerDescription' ||
+		abtest( 'plansDescriptions' ) === 'descendingPriceEagerDescription'
+	);
+
 	return (
-		<div className="plan-features__item">
+		<div className="plan-features__item"
+			{ ...( hoverOnRow && mouseEvents ) }
```

Comment:
Okay, we need a definite resolution on this before merging. I think that while removing the (i) icon might be helpful to reduce the UI clutter ("look at all the small (i) icons on the plans page), it might require additional changes and delay the test.

I would say let's merge this already. :) But I'm waiting for @rralian to give you thumps up on this one, though.